docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,134 @@
 # Cody Ignore
 
-<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
+You can control what context from your codebase is used by Cody in one of two ways:
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+-   Cody Context Filters (Cody Enterprise Only)
+-   Cody Ignore File (All Cody users)
 
-## Enabling Unstable Features
+## Cody Context Filters (Cody Enterprise Only)
+
+Enterprise instances admins can use the Cody Context Filters to determine which repositories Cody is allowed to use as context in its requests to third-party LLMs. This functionality is currently experimental and is only supported under the following conditions:
+
+1.  Having Cody Enterprise license and running a Sourcegraph version `>=5.4.0`
+2.  Using Cody client with Cody context filters support: Cody Web, Cody VS Code Extension, Cody JetBrains Extension. Any other client won't work if Cody context filters are defined.
+3.  Cody client version satisfies the following version constrains: `>=1.20.0` for the VS Code and `>=6.0.0` for the JetBrains. Any earlier client versions won't work if Cody context filters are defined.
+4.  Setting "cody-context-filters-enabled" feature flag to `true`.
+
+Administrators can configure the `cody.contextFilters` field in the site configuration with the repos that they wish Cody to exclude or only include using the following structure:
+
+```json5
+{
+	// Optional. If defined, it must have at least one
+	// field set (either `include` or `exclude`).
+	'cody.contextFilters': {
+		// Optional. If defined, it must have at least one item.
+		include: [
+			{
+				// Required. Should follow the RE2 syntax.
+				repoNamePattern: '^github\\.com\\/sourcegraph\\/.*'
+			}
+		],
+		// Optional. If defined, it must have at least one item.
+		exclude: [
+			{
+				// Required. Should follow the RE2 syntax.
+				repoNamePattern: '.*analytics.*'
+			}
+		]
+	}
+}
+```
+
+### Interaction between include and exclude rules
+
+1.  **`cody.contextFilters` field is not defined**
+
+    By default, there are no restrictions on the repositories Cody can use for context in requests to third-party LLMs.
+    All repositories are included and none are excluded.
+
+2.  **Only `include` rules are specified**
```

Comment:
```suggestion
2. Only `include` rules are specified 

Cody is restricted from using content from repositories whose names match any specified patterns in the include field. Since no `exclude` rules are specified, no repositories are explicitly excluded

<Callout type="note">An `include` field might contain a catch-all pattern (e.g., a `regexp` matching any string). In this case, Cody can access content from any repository, as a single match in the `include` rules is sufficient for inclusion.</Callout>
```